sdk/python/kfp_tekton/tekton.py

```diff
@@ -36,50 +37,67 @@ class AnySequencer(ContainerOp):
-                 image: str = ANY_SEQUENCER_IMAGE):
-        arguments = [
-                    "--namespace",
-                    "$(context.pipelineRun.namespace)",
-                    "--prName",
-                    "$(context.pipelineRun.name)"
-                ]
-        tasks_list = []
-        condition_list = []
-        file_outputs = None
-        for cop in any:
-            if isinstance(cop, dsl.ContainerOp):
-                cop_name = sanitize_k8s_name(cop.name)
-                tasks_list.append(cop_name)
-            elif isinstance(cop, ConditionOperator):
-                condition_list.append(cop)
-        if len(tasks_list) > 0:
-            task_list_str = ",".join(tasks_list)
-            arguments.extend(["--taskList", task_list_str])
-        if statusPath is not None:
-            file_outputs = {"status": statusPath}
-            arguments.extend(["--statusPath", statusPath])
-            if skippingPolicy is not None:
-                assert skippingPolicy == "skipOnNoMatch" or skippingPolicy == "errorOnNoMatch"
-                arguments.extend(["--skippingPolicy", skippingPolicy])
-            if errorPolicy is not None:
-                assert errorPolicy == "continueOnError" or errorPolicy == "failOnError"
-                arguments.extend(["--errorPolicy", errorPolicy])
-
-        conditonArgs = processConditionArgs(condition_list)
-        arguments.extend(conditonArgs)
-
-        super().__init__(
-            name=name,
-            image=image,
-            file_outputs=file_outputs,
-            command="any-task",
-            arguments=arguments,
-        )
+    arguments = [
+                "--namespace",
+                "$(context.pipelineRun.namespace)",
+                "--prName",
+                "$(context.pipelineRun.name)"
+            ]
+    tasks_list = []
+    condition_list = []
+    file_outputs = None
+    for cop in any:
+        if isinstance(cop, dsl.ContainerOp):
+            cop_name = sanitize_k8s_name(cop.name)
+            tasks_list.append(cop_name)
+        elif isinstance(cop, ConditionOperator):
+            condition_list.append(cop)
+    if len(tasks_list) > 0:
+        task_list_str = "\'" + ",".join(tasks_list) + "\'"
+        arguments.extend(["--taskList", task_list_str])
+    if statusPath is not None:
+        file_outputs = '{outputPath: %s}' % statusPath
+        arguments.extend(["--statusPath", file_outputs])
+        if skippingPolicy is not None:
+            assert skippingPolicy == "skipOnNoMatch" or skippingPolicy == "errorOnNoMatch"
+            arguments.extend(["--skippingPolicy", skippingPolicy])
+        if errorPolicy is not None:
+            assert errorPolicy == "continueOnError" or errorPolicy == "failOnError"
+            arguments.extend(["--errorPolicy", errorPolicy])
+    conditonArgs = processConditionArgs(condition_list)
+    arguments.extend(conditonArgs)
+    components._components._outputs_dir = "/tekton/results"
```

Comment:
I think we don't need to overwrite this because this is only display in the description. The actual output path is based on the Tekton variable `$(results.status.path)`

Reply:
I agree, since there's no actual impact. Though seems the description here will overwrite the description I input.